### ISSUE TITLE
Crash on /pilot

### ISSUE THREAD

**Issue by Pocketkid2** [msummers123/Movecraft-3#154]
This happens when I try to pilot a subairship. Signs also dont work. Using latest version of spigot (1.8.8) and latest movecraft beta.

http://pastebin.com/gFChP8dZ

**Comment by TheRealBotcrusher** [comment]
Can you try downloading again? I can't reproduce this so far.

**Comment by Pocketkid2** [comment]
I have confirmed this happens when a craft file is non existent. Please do the following to fix this: Make sure all default craft files (the ones used on the server) are packaged with the plugin/zip download correctly, and if you try to pilot a craft type that doesn't exist, catch the exception and tell the player

**Comment by Pocketkid2** [comment]
Know what, screw the second one, I'll do it myself

**Comment by msummers123** [comment]
Yeah, /pilot does not currently check to see if that craft type actually exists. It's a bug, kind of low priority because players shouldn't be using that anyway. But I'll leave this open so it can be tracked and hopefully fixed someday when I find myself in posession of an excess of time.
